packages/python/plotly/plotly/figure_factory/_scatterplot.py

```diff
@@ -57,10 +57,7 @@ def endpts_to_intervals(endpts):
         # add -inf to intervals
         intervals = [[float("-inf"), endpts[0]]]
         for k in range(length - 1):
-            interval = []
-            interval.append(endpts[k])
-            interval.append(endpts[k + 1])
-            intervals.append(interval)
+            intervals.append([endpts[k],endpts[k + 1]])
```

Comment:
Proposed change:
```python
        # add -inf to intervals
        intervals = [[float("-inf"), endpts[0]]]
        for k in range(length - 1):
            intervals.append([endpts[k], endpts[k + 1]])
```

Reply:
You looked at outdated code, it is already changed in the current version of the pull request.